rotational_PID:
    P : 0.8
    I : 0.001
    D : 0.0
    max_state : 0.5
    min_state : -0.5

translation_PID:
    P : 0.4
    I : 0.01
    D : 0.0
    max_state : 0.15
    min_state : -0.15

robot_stop:
    dist_stop_condition : 0.02
    rotational_stop_condition : 0.02

marker_trace:
    error_s_max : 2.

linear_model_for_target_pos:
    x_slope : 1.
    y_slope : 1.
    x_bias : -0.1
    y_bias : 0.